src/Module/Admin/Site.php

```diff
@@ -551,6 +553,7 @@ public static function content(array $parameters = [])
 			'$private_addons'         => ['private_addons', DI::l10n()->t('Disallow public access to addons listed in the apps menu.'), DI::config()->get('config', 'private_addons'), DI::l10n()->t('Checking this box will restrict addons listed in the apps menu to members only.')],
 			'$disable_embedded'       => ['disable_embedded', DI::l10n()->t('Don\'t embed private images in posts'), DI::config()->get('system', 'disable_embedded'), DI::l10n()->t('Don\'t replace locally-hosted private photos in posts with an embedded copy of the image. This means that contacts who receive posts containing private photos will have to authenticate and load each image, which may take a while.')],
 			'$explicit_content'       => ['explicit_content', DI::l10n()->t('Explicit Content'), DI::config()->get('system', 'explicit_content'), DI::l10n()->t('Set this to announce that your node is used mostly for explicit content that might not be suited for minors. This information will be published in the node information and might be used, e.g. by the global directory, to filter your node from listings of nodes to join. Additionally a note about this will be shown at the user registration page.')],
+			'$proxify_content'        => ['proxify_content', DI::l10n()->t('Proxify external content'), DI::config()->get('system', 'proxify_content'), DI::l10n()->t('Redirect external content via the proxy functionality. This is used for example for some OEmbed accesses and in some other rare cases.')],
```

Comment:
As far as I know, the proxy doesn't redirect, it makes a local copy of the remote resource that is then accessed instead.

Reply:
Well, that's not completely true: https://github.com/friendica/friendica/blob/2021.09-rc/src/Module/Proxy.php#L69-L72

Reply:
Indeed, but it should be clearer in the description.

Reply:
I changed a word.